packages/server/graphql/mutations/removeSlackAuth.ts

```diff
@@ -28,11 +27,11 @@ export default {
     }
 
     // RESOLUTION
-    const teamMemberId = toTeamMemberId(teamId, viewerId)
-    const {authId, error} = await removeSlackAuths(teamMemberId, true)
+    const {authIds, error} = await removeSlackAuths(viewerId, teamId, true)
     if (error) {
       return standardError(error, {userId: viewerId})
     }
+    const authId = authIds![0]
```

Comment:
since `removeSlackAuths` returns a discriminated union on error being null, if you don't destructure immediately, it will give you type safety:

```ts
    const res = await removeSlackAuths(viewerId, teamId, true)
    if (res.error) {
      return standardError(res.error, {userId: viewerId})
    }
    const {authIds} = res
    const authId = authIds[0]
```

Reply:
Ooh well that's wonderful knowledge. Updated & will merge now!